coding-concepts.js

```diff
@@ -8,39 +8,46 @@
 // --------------------1) What will this log?
 
 var cohort = "Alpha 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(''))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The answer will be ["A","l","p","h","a","","2","0","2","2"]
+// b) Verify and explain: By adding the .split() method at the end of a string it will separate each individual character by
+//    the inputted separator and return them in an array.
+/**Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split **/
```

Comment:
Love that you placed references on your answers. These will be great notes to review later.